generators/app/prompts.js

```diff
@@ -115,6 +117,26 @@ module.exports = [
     default: false,
     when: props => props.ui !== 'ionic'
   },
+  {
+    type: 'confirm',
+    name: 'i18n',
+    message: 'Do you want language internationalization support?',
+    default: true
+  },
+  {
+    type: 'checkbox',
+    name: 'i18n-langs',
+    message: 'Which languages do you want to support?',
+    when: props => props.i18n,
+    choices: () => getLangs().map(lang => ({value: lang, name: lang, checked: true}))
+  },
+  {
+    type: 'list',
+    name: 'i18n-default',
+    message: 'Which language are the default(fall back)?',
```

Comment:
`is` the default rather than `are`. Also, there should be a space between `default` and `(`.